### PR TITLE
Added a lorem ipsum generator extension

### PR DESCRIPTION
**Pull request by EdCharbeneau**
This extension method allows you to fill large string properties with Lorem Ipsum placeholder text. This could be useful for Blog posts/comments etc..

Ex: Fill BlogPost.Body with a thousand words
 ```
int wordCount = 1000;
            A.Configure<BlogPost>()
                .Fill(p => p.Body)
                .AsLoremIpsum(wordCount);
```

### CODE REVIEW

**Comment by MisterJames** [comment]
Whoa. That's sexy. :)

I'll review this over the weekend!  Thanks @EdCharbeneau!

File: src/GenFu/ValueGenerators/Lorem/Lorem.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            return GetRandomValue(ResourceLoader.Data(Properties.Lorem));
+        }
+        public static string LoremIpsum(int wordCount)
+        {
+            var words = new List<string>();
+            for (int i = 0; i < wordCount; i++)
+            {
+                words.Add(GetRandomValue(ResourceLoader.Data(Properties.Lorem)));
+            }
+            return string.Join(" ", words);
```
Comment (MisterJames) [inline src/GenFu/ValueGenerators/Lorem/Lorem.cs]: Do you think it should start with a capital an end with a period?

**Comment by MisterJames** [comment]
Other than the fact that we really need to drop those lock files this looks good. 

I think it would be worth using sentence case, though. 

> Lorem ipsum dolor sit amet.

Thoughts?

**Comment by EdCharbeneau** [comment]
Awesome suggestion.  I think it should be optional, but on by default. Do
you want to merge now, or let me make the addition first?

**Comment by EdCharbeneau** [comment]
Actually, that gave me some even bigger ideas. Let me fix this up first.

**Comment by MisterJames** [comment]
Oh sure, take your time! 

...but you didn't need to close the PR, you can just keep adding commits to it (for future pull requests).

**Pull request closed without merging**